use-timescale/extensions/index.md

```diff
@@ -13,50 +13,120 @@ supported extensions:
+| [jsonb_plperl][jsonb_plperl]                     | Transform between jsonb and plperl                                     | -                              | 
+| [lo][lo]                                         | Large object maintenance                                               | -                              |
+| [ltree][ltree]                                   | Data type for hierarchical tree-like structures                        | -                              |
+| [moddatetime][moddatetime]                       | Functions for tracking last modification time                          | -                              |
+| [old_snapshot][old_snapshot]                     | Utilities in support of `old_snapshot_threshold`                       | -                              |
+| [pg_freespacemap][pg_freespacemap]               | Examine the free space map (FSM)                                       | -                              |
+| [pg_prewarm][pg_prewarm]                         | Prewarm relation data                                                  | -                              |
+| [pg_stat_statements][pg_stat_statements]         | Track execution statistics of all SQL statements executed              | All                            |
+| [pg_trgm][pg_trgm]                               | Text similarity measurement and index searching based on trigrams      | -                              |
+| [pg_visibility][pg_visibility]                   | Examine the visibility map (VM) and page-level visibility info         | -                              |
+| [pgai][pgai]                                     | Helper functions for AI workflows                                      | All                            |
+| [pgaudit][pgaudit]                               | Detailed session and/or object audit logging                           | -                              |
+| [pgcrypto][pgcrypto]                             | Cryptographic functions                                                | -                              |
+| [pgpcre][pgpcre]                                 | Perl-compatible RegEx                                                  | -                              |
+| [pgrouting][pgrouting]                           | Geospatial routing functionality                                       | -                              |
+| [pgrowlocks][pgrowlocks]                         | Show row-level locking information                                     | -                              |
+| [pgstattuple][pgstattuple]                       | Obtain tuple-level statistics                                          | -                              |
+| [pgvector][pgvector]                             | Vector similarity search for PostgreSQL                                | AI and vector                  |
```

Comment:
it's called `vector` in the extensions list

Reply:
See comment above about ai